# invalid command received
string command_received
